gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -3236,19 +3236,40 @@ cmd VkResult vkResetEvent(
   return ?
 }
 
+@override
+@custom
+@no_replay
+cmd void RecreateQueryPool(
```

Comment:
Nit: It might make sense from a cleanliness POV to move non-Vulkan APIs to their own API file.